### DISCUSSION TITLE
Robust options of fvSolution for DASimpleFOAM

### DISCUSSION THREAD

**Discussion by justAPhDStudent** [mdolab/dafoam#203] [General]
Hello,

I have been just playing with the DASimpleFOAM & DATurboFOAM solvers for some toy cases. I tried 'wind tunnel's boxes of several sizes to see some significant effect on the convergence. I know there should be some minimum criterion, in terms of the characteristics dimension D. 
For the DASimpleFOAM case, there is lack of convergence in pressure field & the z-component of velocity. The problem was even attempted with sufficient mesh criterion on 120 processes. It seems may be the BC is playing some role. Can you suggest some solvers or modifications when attempting a solve on an arbitrary 3D aerodynamics DASimpleFOAM solve. I'm just trying a car rear wing for fun. 

For DATurboFOAM, I have been playing with a ship propellor. Of course I tried an rpm of 100 instead of the 10 for NREL case. I don't know if this causing the convergence issue for this. Here the overall solution convergence is low. Can you suggest some info or guide me with some options to play with for this DATurboFOAM tutorial case. May be making the pressure outlet to $internalField can be an option. Or using PCG to solve for pressure also can make a difference. Do you have some suggestions for improvements. 

Thanks very much in advance

**Comment by friedenhe** [comment]
Check the tutorials, we have quite a few 3D wing aero cases with DASimpleFoam. We use the best options we think for these cases. Regarding the DATurboFoam, make sure you check every BC setup, especially for U.

**Comment by justAPhDStudent** [comment]
Thank you for your prompt guidance. As specifically for DASimpleFOAM, there are only two different tutorials: NACA incompressible & JBC hull. All other wing tutorials are for compressible solvers DARhoSimpleFOAM, whose pressure BC is not the same as that of DASimpleFOAM. I point this out because the field p is the one I'm facing difficulty converging. I was hoping some guidance on incompressible solvers, as the tutorials you suggested are mostly compressible solvers.
Thank you in advance.

**Reply by friedenhe** [reply]
You can use the BC types used in the NACA0012 incompressible case for a 3D wing. Without more details, it is hard to know what went wrong with your DASimpleFoam simulations.
